src/filters/FiltersActions.js

```diff
@@ -86,6 +92,32 @@ export const setSpecificBreakdown = (specificBreakdown) => {
   };
 };
 
+const parseFilterRanges = (ranges, filterState) => {
+
+  const newFilters = _.reduce(ranges, (accum, value, aggKey) => {
```

Comment:
`ranges` is an object here, and I thought it would be clearer with a named lodash function instead of a loop.